src/services/conseillers/controllers/getConseillersPourLaCoopMediation.ts

```diff
@@ -0,0 +1,295 @@
+    updatedAt: string;
+    updatedBy: string;
+  }>;
+  prenom: string;
+  statut: string;
+  structure: Readonly<Record<string, boolean | Date | Object | string>>;
+  subordonnes: ReadonlyArray<{
+    idPG: number;
+  }>;
+}>;
+
+async function retournerDesConseillersRecrutesParOrdreCroissant(
+  app: Application,
+  email: string = '',
+  idPg: number = 0,
+  limit: number = 10,
+  page: number = 0,
+): Promise<ReadonlyArray<ConseillerPourLaCoopMediation>> {
+  let match = {};
+
+  if (email !== '') {
+    match = { email };
+  } else if (idPg !== 0) {
+    match = { idPG: idPg };
+  }
+
+  const conseillers = await app.service(service.conseillers).Model.aggregate([
+    {
+      $match: {
+        statut: 'RECRUTE',
+        ...match,
+      },
+    },
+    {
+      $sort: {
+        idPG: 1,
+      },
+    },
+    {
+      $skip: page > 0 ? (page - 1) * limit : 0,
+    },
+    {
+      $limit: limit,
+    },
+    {
+      $lookup: {
+        from: 'permanences',
+        let: {
+          idConseiller: '$_id',
+          structureId: '$structureId',
+        },
+        as: 'permanences',
+        pipeline: [
+          {
+            $match: {
+              $expr: {
+                $in: ['$$idConseiller', '$conseillers'],
+              },
+            },
+          },
+          {
+            $project: {
+              _id: 1,
+              adresse: 1,
+              conseillers: 1,
+              conseillersItinerants: 1,
+              email: 1,
+              estStructure: 1,
+              horaires: 1,
+              lieuPrincipalPour: 1,
+              location: 1,
+              nomEnseigne: 1,
+              numeroTelephone: 1,
+              siret: 1,
+              siteWeb: 1,
+              structure: '$$structureId',
+              typeAcces: 1,
+              updatedAt: 1,
+              updatedBy: 1,
+            },
+          },
+        ],
+      },
+    },
+    {
+      $lookup: {
+        localField: 'structureId',
+        from: 'structures',
+        foreignField: '_id',
+        as: 'structure',
+      },
+    },
+    {
+      // $lookup: {
+      //   from: 'conseillers',
+      //   let: {
+      //     idsCoordinateur: '$coordinateurs.id',
+      //   },
+      //   as: 'subordonnes',
+      //   pipeline: [
+      //     {
+      //       $match: {
+      //         $expr: {
+      //           $in: ['$_id', '$$idsCoordinateur'],
+      //         },
+      //       },
+      //     },
+      //     {
+      //       $project: {
+      //         _id: 0,
+      //         idPG: 1,
+      //       },
+      //     },
+      //   ],
+      // },
+      $lookup: {
+        localField: 'coordinateurs.id',
+        from: 'conseillers',
+        foreignField: '_id',
+        as: 'subordonnes',
+      },
+    },
+    {
+      $lookup: {
+        from: 'misesEnRelation',
+        let: {
+          idConseiller: '$_id',
+        },
+        as: 'misesEnRelation',
+        pipeline: [
+          {
+            $match: {
```

Comment:
peut etre ici faire un in des statut : 

`statut: {'$in': ['nouvelle_rupture', 'finalisee_rupture', 'finalisee', terminee]}`